ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallFooter.kt

```diff
@@ -29,28 +30,38 @@ import androidx.compose.ui.tooling.preview.Preview
 fun PaywallFooter(
     options: PaywallOptions,
     condensed: Boolean = false,
-    mainContent: @Composable (PaddingValues) -> Unit,
+    mainContent: @Composable ((PaddingValues) -> Unit)? = null,
```

Comment:
I am a bit confused about this change. When would someone not want to set something? This is a footer anywyas

Reply:
Right... The main reason for this change was that the `Column` composable inside has a `fillMaxSize` modifier, which we don't want when we want to render only the footer view. In native android, we probably want people to pass us a composable, but when using the `PaywallFooterView`, we can't easily pass composables as a parameter to the view (most people using the view are probably not using compose).

An alternative to this would be to add a parameter, `useMaxSize` or something like that that tells us whether this composable should use all available space. I kinda didn't want to add another parameter if possible, but it might be less error prone than making the content nullable... wdyt?

Reply:
Hmm, I see... I am seeing now that `mainContent` is a confusing parameter. It makes it feel as if it's the content of the Footer, and not whatever is _behind_ the Footer

I am honestly leaning towards creating two views, the current `PaywallFooter` should be something like `PaywallFooterWithScreenContent` and a new `PaywallFooter` shouldn't paint anything else and not accept a `mainContent`

We can discuss this in our weekly sync I think